.github/workflows/wcag_test.yml

```diff
@@ -34,4 +36,8 @@ jobs:
       # `npm rebuild` will run all those post-install scripts for us.
       - run: npm rebuild
       - run: ${{ inputs.build_script }}
+        env:
+          MAPBOX_API_KEY: ${{ secrets.MAPBOX_API_KEY }}
       - run: npm run wcag
+        env:
+          MAPBOX_API_KEY: ${{ secrets.MAPBOX_API_KEY }}
```

Comment:
do we need the API key in both steps? or is just having it in the build step sufficient?

Reply:
in the build step should be sufficient, but I was thinking potentially this step in the future might want access to the env as well. I could remove it for now though